### DISCUSSION TITLE
【InconsistentTablet】如何手动修复至健康状态

### DISCUSSION THREAD

**Discussion by iplaypi** [apache/doris#9674] [A - General / Q&A]
背景：
5台be节点，由于故障挂掉3-4台，重启后，某个库会出现一些InconsistentTablet，无法自动恢复。
在文档中也没找到手动恢复的方法：https://doris.apache.org/zh-CN/administrator-guide/operation/tablet-repair-and-balance.html#%E5%90%8D%E8%AF%8D%E8%A7%A3%E9%87%8A 。
查询这个表的数据时，针对REPLACE类型的字段，会出现值的变化，同一条数据每次查询结果都不一样，像是查到了历史各个版本的值。
![image](https://user-images.githubusercontent.com/13584796/169191400-7a4b7369-4db1-4631-8aa3-624e02d85469.png)

做法：
现在的做法是尝试新建库，把数据迁移到新库，数据tablet就恢复正常了（复制到当前库的其它表不行，仍旧不一致）。

疑问：
1、UnhealthyTablet可以被Doris自动恢复（全部），但是InconsistentTablet无法恢复（总是有部分）。
2、这种情况怎么可以手动解决？

**Comment by hf200012** [comment]
InconsistentTablet 这个可以忽略，不要关系，只要不出现UnhealthyTablet
这个问题你可以看看：https://doris.apache.org/zh-CN/faq/sql-faq.html#q4-unique-key-%E6%A8%A1%E5%9E%8B%E6%9F%A5%E8%AF%A2%E7%BB%93%E6%9E%9C%E4%B8%8D%E4%B8%80%E8%87%B4
